Annex60/Fluid/Sensors/TemperatureTwoPort.mo

```diff
@@ -63,8 +64,9 @@ equation
   end if;
   // Output signal of sensor
   if dynamic then
-    if enableThermalLosses then
-      der(T) = (TMed-T)*k*tauInv + (TAmb-T)*tauLossInv;
+    if transferHeat then
+      // Note that Dymola will remove the term (TAmb-T)*tauHeaTraInv if tauHeaTraInv=0.
```

Comment:
With the current implementation tauHeaTraInv should never be zero.
Also, I think this is only true when setting  ```Evaluate=true```, although I did not verify this.